components/icu/src/lib.rs

```diff
@@ -83,6 +83,7 @@
 //! [`Locale`]: crate::locid::Locale
 //! [`SymbolsV1`]: crate::decimal::provider::DecimalSymbolsV1
 
+#![warn(missing_docs)]
```

Comment:
Question: Is there a way to make this a global Clippy lint that is disabled for certain crates, rather than enabled for certain crates (opt-out rather than opt-in)?

Reply:
Note: If this were possible, I would prefer if we could still exclude crates under *experimental* from the flag by default.

Reply:
so technically this isn't a clippy lint, but we can absolutely update our `cargo clippy` invocation (in config.toml and GHA) to have a `-Wmissing_docs`

Reply:
It is not possible to do per folder opt outs that way. We can just per crate opt out with the attribute

Reply:
Would such per-directory opt-outs be enabled by https://github.com/rust-lang/cargo/issues/5034?

Reply:
Unlikely